terraform/aws-accounts/cloud-platform-aws/vpc/eks/cluster.tf

```diff
@@ -35,9 +35,9 @@ locals {
   }
   # To manage different cluster versions
   cluster_version = {
-    live    = "1.21"
-    live-2  = "1.21"
-    manager = "1.21"
+    live    = "1.22"
+    live-2  = "1.22"
+    manager = "1.22"
     default = "1.21"
```

Comment:
We should change the default to 1.22 also so all test clusters will be the same version